api/app.py

```diff
@@ -166,6 +201,62 @@ def signup():
     else:
         return jsonify("")
 
+def send_mail(user):
+    token=user.get_token()
+    msg=Message('Password Reset Request',recipients=[user.email], sender='xchangeuw@outlook.com')
+    print(url_for('forgot_password_success', token=token,_external=True).replace("http://127.0.0.1:5000", "http://127.0.0.1:3000"))
+    msg.body=f''' To reset your password, please follow the link below:
+    
+    {url_for('forgot_password_success', token=token,_external=True).replace("http://127.0.0.1:5000", "http://127.0.0.1:3000")}
+    If you didn't send a password reset request, please ignore this message.
+
+    '''
+
+    mail.send(msg)
+
+@app.route('/forgot_password', methods=['GET','POST'])
+#Need to use UserBuilder, or do the same salting protocol as in UserBuilder
```

Comment:
There is not other mention of UserBuilder from the new code added in this PR, Is this no longer required ? I do not see it used in this route.

Reply:
No longer needed